docs/fundamentals/flow/health-check.md

```diff
@@ -330,4 +330,47 @@ curl http://localhost:12345
 And you will get a valid empty response indicating the Gateway's ability to serve.
 ```json
 {}
+```
+
+## Use jina ping to do health checks
+
+Once a Flow is running, you can use `jina ping` CLI (TODO: Add link to documentation) to run readiness check of the complete Flow or of individual Executors or Gateway.
+
+Let's start a Flow in the terminal:
+
+```python
+from jina import Flow
+
+with Flow(protocol='grpc', port=12345).add(port=12346) as f:
+    f.block()
+```
+
+We can check the readiness of the Flow:
+
+```bash
+jina ping flow grpc://localhost:12345
+```
```

Comment:
maybe we can say here that it check that the flow is ready by sending a dry run requests ?

Reply:
If this is added, is another scope, because this only explains the `jina ping`. Details may belong up in the document